ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -27,13 +27,17 @@ typedef NS_ENUM(NSInteger, DeviceType) {
     DeviceTypeTablet,
     DeviceTypeTv,
     DeviceTypeDesktop,
+    DeviceTypeHeadset,
     DeviceTypeUnknown
 };
 
-#define DeviceTypeValues [NSArray arrayWithObjects: @"Handset", @"Tablet", @"Tv", @"Desktop", @"unknown", nil]
+#define DeviceTypeValues [NSArray arrayWithObjects: @"Handset", @"Tablet", @"Tv", @"Desktop", @"Headset", @"unknown", nil]
 
-#if !(TARGET_OS_TV)
+#if !(TARGET_OS_TV) && !(TARGET_OS_VISION)
 @import CoreTelephony;
+#endif
+
+#if TARGET_OS_VISION && !TARGET_OS_TV
```

Comment:
I don't understand this conditional. Isn't "if it is visionos and it is not tvos" the same as saying "if it is visionos"?
Stated differently I think the second conditional will always be true if the first is true? 

```suggestion
#if TARGET_OS_VISION
```

...but I could be missing something!